### PR TITLE
Replace a ton of manual checks with can_act calls

### PR DESCRIPTION
**Pull request by BatElite**
[cleanliness]
## About the PR 
A lot of things in a lot of places do checks that amount (or should/nearly amount) to calling ```can_act```.  This PR makes them use ```can_act``` instead. Did you know that there were 24 results for searching "can_act(" previously (including the proc definition)? This PR bumps that up to 230.

The third commit focuses on all the checks that are incomplete compared to what ```can_act``` does. Often these don't check one of the status effects, or they don't have an isalive/stat check. I've tried to only replace those that made sense.

Whoever will review this might notice that there's several copies of near-identical procs being poked. Moving those to parents is for a later PR. I've also not fixed the galaxybraining going on in frenzy.dm that stuns the user and then immediately aborts because the user is stunned.

misc notes:
-Some of the files poked are adventure zone stuff. I'm pretty sure none of them should be meaningfully altered.
-I've tested it a bit and the whole thing compiles without any issues, but I don't know how I'd check everything in here.
-Not all of this stuff might actually be in use.

## Why's this needed? 
Cleeeen code, at least somewhat. 

At least one check was partially borked, and a good number of places weren't checking everything they should.
(Maybe can_act should check for pinning too, but that's for another PR)

### CODE REVIEW

File: code/obj/item/clothing/injector_mask_belt.dm [hunk 1]
```diff
@@ -414,7 +414,7 @@ There's A LOT of duplicate code here, which isn't ideal to say the least. Should
 		return 1
 
 	check_trigger(mob/M)
-		if(M.getStatusDuration("stunned") || M.getStatusDuration("paralysis") || M.getStatusDuration("weakened") || isunconscious(M)) return 1
+		if(!can_act(M, 0)) return 1
```
Comment (BatElite) [inline code/obj/item/clothing/injector_mask_belt.dm:417]: I want to point out that as it is the new code will trigger on death when the old one didn't, but I don't expect it to be an important difference. Dying without going unconscious first is somewhat of a tall order anyway.

**Comment by Tarmunora** [comment]
Haven't read through yet, but I'd be concerned about explicit checks for paralyzed/KOed being changed to include stunned/weakened. Stuff like talking and table-surgery comes to mind

File: code/modules/medical/surgery_procs.dm [hunk 2]
```diff
@@ -56,7 +56,7 @@ var/global/list/chestitem_whitelist = list(/obj/item/gnomechompski, /obj/item/gn
 		else if (patient == surgeon)
 			return 3.5 // surgery is okay but hurts more
 
-	else if (locate(/obj/table, patient.loc) && (patient.getStatusDuration("paralysis") || patient.stat)) // is the patient on a table and paralyzed or dead?
+	else if (locate(/obj/table, patient.loc) && !can_act(patient, 0)) // is the patient on a table and incapacitated or dead?
```
Comment (Tarmunora) [inline code/modules/medical/surgery_procs.dm:59]: Note: this allows table surgery on stunned but not KOed people

File: code/obj/critter/bee.dm [hunk 3]
```diff
@@ -101,7 +101,7 @@
 			if (!istype(M)) return
 			if (prob(20))
 				return CritterAttack(M)
-			if (M.stat || M.getStatusDuration("paralysis"))
+			if (!can_act(M, 0))
```
Comment (Tarmunora) [inline code/obj/critter/bee.dm:104]: Similarly here (and throughout this file) we have 'is-asleep' checks being changed to 'is-stunned'

File: code/obj/item.dm [hunk 4]
```diff
@@ -615,7 +615,7 @@
 /obj/item/MouseDrop(atom/over_object, src_location, over_location, over_control, params)
 	..()
 
-	if (usr.stat || usr.restrained() || !can_reach(usr, src) || usr.getStatusDuration("paralysis") || usr.sleeping || usr.lying || isAIeye(usr) || isAI(usr) || isrobot(usr) || isghostcritter(usr) || (over_object && over_object.event_handler_flags & NO_MOUSEDROP_QOL))
+	if (!can_act(usr, 1) || !can_reach(usr, src) || usr.sleeping || usr.lying || isAIeye(usr) || isAI(usr) || isrobot(usr) || isghostcritter(usr) || (over_object && over_object.event_handler_flags & NO_MOUSEDROP_QOL))
```
Comment (Tarmunora) [inline code/obj/item.dm:618]: Mechanical change here

File: code/obj/item/brain.dm [hunk 5]
```diff
@@ -71,7 +71,7 @@
 	if(!(user.zone_sel.selecting == ("head")) || !ishuman(M))
 		return ..()
 
-	if(!(locate(/obj/machinery/optable, M.loc) && M.lying) && !(locate(/obj/table, M.loc) && (M.getStatusDuration("paralysis") || M.stat)))
+	if(!(locate(/obj/machinery/optable, M.loc) && M.lying) && !(locate(/obj/table, M.loc) && !can_act(M, 0)))
```
Comment (Tarmunora) [inline code/obj/item/brain.dm:74]: table surgery check again

File: code/obj/item/chem_pill_bottle.dm [hunk 6]
```diff
@@ -120,7 +120,7 @@
 			return ..()
 
 	MouseDrop_T(atom/movable/O as obj, mob/user as mob)
-		if (user.restrained() || user.getStatusDuration("paralysis") || user.sleeping || user.stat || user.lying)
+		if (!can_act(user, 1) || user.sleeping || user.lying)
```
Comment (Tarmunora) [inline code/obj/item/chem_pill_bottle.dm:123]: mechanical change

File: code/obj/item/clothing/gimmick.dm [hunk 7]
```diff
@@ -74,7 +74,7 @@
 			return 0
 		if (ishuman(user))
 			var/mob/living/carbon/human/H = user
-			if (istype(H.w_uniform, /obj/item/clothing/under/gimmick/owl) && !(user.stat || user.getStatusDuration("paralysis")))
+			if (istype(H.w_uniform, /obj/item/clothing/under/gimmick/owl) && can_act(user, 0))
```
Comment (Tarmunora) [inline code/obj/item/clothing/gimmick.dm:77]: mechanical change

File: code/obj/item/generic_box.dm [hunk 8]
```diff
@@ -339,7 +339,7 @@
 						I.set_loc(T)
 
 	MouseDrop_T(atom/movable/O as obj, mob/user as mob)
-		if (user.restrained() || user.getStatusDuration("paralysis") || user.sleeping || user.stat || user.lying)
+		if (!can_act(user, 1) || user.sleeping || user.lying)
```
Comment (Tarmunora) [inline code/obj/item/generic_box.dm:342]: mechanical change

File: code/obj/item/gun/kinetic.dm [hunk 9]
```diff
@@ -1197,7 +1197,7 @@
 		..()
 
 	MouseDrop(atom/over_object, src_location, over_location, params)
-		if (usr.stat || usr.restrained() || !can_reach(usr, src) || usr.getStatusDuration("paralysis") || usr.sleeping || usr.lying || isAIeye(usr) || isAI(usr) || isghostcritter(usr))
+		if (!can_act(usr, 1) || !can_reach(usr, src) || usr.sleeping || usr.lying || isAIeye(usr) || isAI(usr) || isghostcritter(usr))
```
Comment (Tarmunora) [inline code/obj/item/gun/kinetic.dm:1200]: mechanical change

File: code/obj/item/misc_wizard_stuff.dm [hunk 10]
```diff
@@ -40,7 +40,7 @@
 
 /obj/item/teleportation_scroll/Topic(href, href_list)
 	..()
-	if (usr.getStatusDuration("paralysis") || !isalive(usr) || usr.restrained())
+	if (!can_act(usr, 1))
```
Comment (Tarmunora) [inline code/obj/item/misc_wizard_stuff.dm:43]: mechanical change

File: code/obj/item/tool/weldingtool.dm [hunk 11]
```diff
@@ -79,7 +79,7 @@
 								return
 						else return ..()
 				else
-					if (!(locate(/obj/machinery/optable, M.loc) && M.lying) && !(locate(/obj/table, M.loc) && (M.getStatusDuration("paralysis") || M.stat)) && !(M.reagents && M.reagents.get_reagent_amount("ethanol") > 10 && M == user))
+					if (!(locate(/obj/machinery/optable, M.loc) && M.lying) && !(locate(/obj/table, M.loc) && !can_act(M, 0)) && !(M.reagents && M.reagents.get_reagent_amount("ethanol") > 10 && M == user))
```
Comment (Tarmunora) [inline code/obj/item/tool/weldingtool.dm:82]: mechanical change

File: code/obj/mopbucket.dm [hunk 12]
```diff
@@ -52,7 +52,7 @@
 	src.transfer_all_reagents(over_object, usr)
 
 /obj/mopbucket/MouseDrop_T(atom/movable/O as mob|obj, mob/user as mob)
-	if (!in_range(user, src) || user.restrained() || user.getStatusDuration("paralysis") || user.sleeping || user.stat || user.lying || user.buckled)
+	if (!in_range(user, src) || !can_act(user, 1) || user.sleeping || user.lying || user.buckled)
```
Comment (Tarmunora) [inline code/obj/mopbucket.dm:55]: mechanical change

File: code/obj/rack.dm [hunk 13]
```diff
@@ -57,7 +57,7 @@
 		return 0
 
 /obj/rack/MouseDrop_T(obj/O as obj, mob/user as mob)
-	if (!isitem(O) || !in_range(user, src) || !in_range(user, O) || user.restrained() || user.getStatusDuration("paralysis") || user.sleeping || user.stat || user.lying)
+	if (!isitem(O) || !in_range(user, src) || !in_range(user, O) || !can_act(user, 1) || user.sleeping || user.lying)
```
Comment (Tarmunora) [inline code/obj/rack.dm:60]: mechanical change

File: code/obj/storage/large_storage_parent.dm [hunk 14]
```diff
@@ -296,7 +296,7 @@
 
 	MouseDrop_T(atom/movable/O as mob|obj, mob/user as mob)
 		var/turf/T = get_turf(src)
-		if (!in_range(user, src) || !in_range(user, O) || user.restrained() || user.getStatusDuration("paralysis") || user.sleeping || user.stat || user.lying || isAI(user))
+		if (!in_range(user, src) || !in_range(user, O) || !can_act(user, 1) || user.sleeping || user.lying || isAI(user))
```
Comment (Tarmunora) [inline code/obj/storage/large_storage_parent.dm:299]: mechanical change

File: code/obj/table.dm [hunk 15]
```diff
@@ -321,7 +321,7 @@
 			return 0
 
 	MouseDrop_T(atom/O, mob/user as mob)
-		if (!in_range(user, src) || !in_range(user, O) || user.restrained() || user.getStatusDuration("paralysis") || user.sleeping || user.stat || user.lying)
+		if (!in_range(user, src) || !in_range(user, O) || !can_act(user, 1) || user.sleeping || user.lying)
```
Comment (Tarmunora) [inline code/obj/table.dm:324]: mechanical change

File: code/procs/mob_procs.dm [hunk 16]
```diff
@@ -178,7 +178,7 @@
 	return 1
 
 /mob/living/carbon/human/sight_check(var/consciousness_check = 0)
-	if (consciousness_check && (src.hasStatus("paralysis") || src.sleeping || src.stat || src.hibernating))
+	if (consciousness_check && (!can_act(src, 0) || src.sleeping || src.hibernating))
```
Comment (Tarmunora) [inline code/procs/mob_procs.dm:181]: Being stunned should not blind

File: code/procs/mob_procs.dm [hunk 17]
```diff
@@ -194,7 +194,7 @@
 	return 1
 
 /mob/living/critter/sight_check(var/consciousness_check = 0)
-	if (consciousness_check && (src.getStatusDuration("paralysis") || src.sleeping || src.stat))
+	if (consciousness_check && (!can_act(src, 0) || src.sleeping))
```
Comment (Tarmunora) [inline code/procs/mob_procs.dm:197]: being stunned should not blind

File: code/procs/mob_procs.dm [hunk 18]
```diff
@@ -320,7 +320,7 @@
 	return 1
 
 /mob/living/carbon/human/hearing_check(var/consciousness_check = 0)
-	if (consciousness_check && (src.stat || src.getStatusDuration("paralysis") || src.sleeping))
+	if (consciousness_check && (!can_act(src, 0) || src.sleeping))
```
Comment (Tarmunora) [inline code/procs/mob_procs.dm:323]: being stunned should not deafen

File: code/procs/mob_procs.dm [hunk 19]
```diff
@@ -336,7 +336,7 @@
 			.= 0
 
 /mob/living/silicon/hearing_check(var/consciousness_check = 0)
-	if (consciousness_check && (src.getStatusDuration("paralysis") || src.sleeping || src.stat))
+	if (consciousness_check && (!can_act(src, 0) || src.sleeping))
```
Comment (Tarmunora) [inline code/procs/mob_procs.dm:339]: being stunned should not deafen

**Review by Tarmunora — CHANGES_REQUESTED** [review]
A number more places where checks for only stat/paralysis (i.e.: mob is knocked out) are replaced with can_act. Some might be fine as can_act, but others should not be changed to can_act, notably the hearing/sight checks and table surgery.

**Comment by BatElite** [comment]
I was going to revert the surgery-related checks so there was something happening in this PR (and then look at the other cases individually) but now I'm not so sure - can_act does a ```stat``` check, so if that's how being asleep is checked then shouldn't surgery be fine? I've done surgery on a monkey with N2O internals on this branch earlier and that worked.

Maybe weakened and stunned aren't necessarily appropriate in that case, I more or less reasoned that "unable to take any actions" included being a problem for a surgeon. :P 

(Will look into the blinding & bee checks though, tomorrow)

File: code/WorkInProgress/HaineWhatever.dm [hunk 20]
```diff
@@ -1185,7 +1185,7 @@ var/list/special_parrot_species = list("ikea" = /datum/species_info/parrot/kea/i
 	cooldown = 100
 
 	ability_allowed()
-		if (!the_mob || the_mob.stat || the_mob.getStatusDuration("paralysis"))
+		if (!the_mob || !can_act(the_mob, 0))
```
Comment (Tarmunora) [inline code/WorkInProgress/HaineWhatever.dm:1188]: mechanical change

File: code/WorkInProgress/cryospawn.dm [hunk 21]
```diff
@@ -188,11 +188,11 @@
 			boutput(L, "<b>You have to be alive to enter cryogenic storage!</b>")
 			boutput(user, "<b>You can't put someone in cryogenic storage if they aren't alive!</b>")
 			return 0
-		if (L.stat || L.restrained() || L.getStatusDuration("paralysis") || L.sleeping)
+		if (!can_act(L, 1) || L.sleeping)
 			boutput(L, "<b>You can't enter cryogenic storage while incapacitated!</b>")
 			boutput(user, "<b>You can't put someone in cryogenic storage while they're incapacitated!</b>")
 			return 0
-		if (user && (user.stat || user.restrained() || user.getStatusDuration("paralysis") || user.sleeping))
+		if (user && (!can_act(user, 1) || user.sleeping))
```
Comment (Tarmunora) [inline code/WorkInProgress/cryospawn.dm:195]: mechanical change

File: code/WorkInProgress/tug.dm [hunk 22]
```diff
@@ -10,7 +10,7 @@
 	mats = 10
 
 	MouseDrop_T(var/atom/movable/C, mob/user)
-		if (!in_range(user, src) || !in_range(user, C) || user.restrained() || user.getStatusDuration("paralysis") || user.sleeping || user.stat || user.lying)
+		if (!in_range(user, src) || !in_range(user, C) || !can_act(user, 1) || user.sleeping || user.lying)
```
Comment (Tarmunora) [inline code/WorkInProgress/tug.dm:13]: mechanical change

File: code/WorkInProgress/tug.dm [hunk 23]
```diff
@@ -46,7 +46,7 @@
 		..()
 		var/turf/T = get_turf(over_location)
 		var/mob/user = usr
-		if (!user || !(in_range(user, src) || user.loc == src) || !in_range(src, over_object) || user.restrained() || user.getStatusDuration("paralysis") || user.sleeping || user.stat || user.lying)
+		if (!user || !(in_range(user, src) || user.loc == src) || !in_range(src, over_object) || !can_act(user, 1) || user.sleeping || user.lying)
```
Comment (Tarmunora) [inline code/WorkInProgress/tug.dm:49]: mechanical change

File: code/WorkInProgress/tug.dm [hunk 24]
```diff
@@ -240,7 +240,7 @@
 				M.set_loc(src.loc)
 
 	MouseDrop_T(var/atom/movable/C, mob/user)
-		if (!in_range(user, src) || !in_range(user, C) || user.restrained() || user.getStatusDuration("paralysis") || user.sleeping || user.stat || user.lying)
+		if (!in_range(user, src) || !in_range(user, C) || !can_act(user, 1) || user.sleeping || user.lying)
```
Comment (Tarmunora) [inline code/WorkInProgress/tug.dm:243]: mechanical change

File: code/datums/abilities/misc/hunter_gearspawn.dm [hunk 25]
```diff
@@ -39,7 +39,7 @@
 
 		var/mob/living/M = owner
 
-		if (M == null || !ishuman(M) || !isalive(M) || M.getStatusDuration("paralysis") > 0 || !transform)
```
Comment (Tarmunora) [inline code/datums/abilities/misc/hunter_gearspawn.dm:42]: mechanical change

File: code/datums/abilities/misc/werewolf_transform.dm [hunk 26]
```diff
@@ -39,7 +39,7 @@
 
 		var/mob/living/M = owner
 
-		if (M == null || !ishuman(M) || !isalive(M) || M.getStatusDuration("paralysis") > 0 || !transform)
```
Comment (Tarmunora) [inline code/datums/abilities/misc/werewolf_transform.dm:42]: mechanical change

File: code/datums/abilities/werewolf/werewolf_transform.dm [hunk 27]
```diff
@@ -40,7 +40,7 @@
 
 		var/mob/living/M = owner
 
-		if (M == null || !ishuman(M) || M.stat != 0 || M.getStatusDuration("paralysis") || !transform)
```
Comment (Tarmunora) [inline code/datums/abilities/werewolf/werewolf_transform.dm:43]: mechanical change

File: code/datums/abilities/wizard/phaseshift.dm [hunk 28]
```diff
@@ -35,7 +35,7 @@
 	if (!isturf(H.loc))
 		H.show_text("You can't seem to turn incorporeal here.", "red")
 		return
-	if (H.stat || H.getStatusDuration("paralysis") > 0)
+	if (!can_act(H, 0))
```
Comment (Tarmunora) [inline code/datums/abilities/wizard/phaseshift.dm:38]: ***critical mechanical change***

File: code/datums/abilities/wizard/teleport.dm [hunk 29]
```diff
@@ -22,7 +22,7 @@
 
 // These two procs were so similar that I combined them (Convair880).
 /mob/proc/teleportscroll(var/effect = 0, var/perform_check = 0, var/obj/item_to_check = null, var/datum/targetable/spell/teleport/spell)
-	if (src.getStatusDuration("paralysis") || !isalive(src))
```
Comment (Tarmunora) [inline code/datums/abilities/wizard/teleport.dm:25]: ***critical mechanical change***

File: code/mob/melee_attack_procs.dm [hunk 30]
```diff
@@ -479,7 +479,7 @@
 #undef DISARM_WITH_ITEM_TEXT
 
 /mob/proc/check_block(ignoreStuns = 0) //am i blocking?
-	if (ignoreStuns || (isalive(src) && !getStatusDuration("paralysis")))
+	if (ignoreStuns || can_act(src, 0))
```
Comment (Tarmunora) [inline code/mob/melee_attack_procs.dm:482]: mechanical change - this was specifically changed in past from stun+weak+para check to just para

File: code/modules/antagonists/changeling/abilities/changeling.dm [hunk 31]
```diff
@@ -333,7 +333,7 @@
 		var/datum/abilityHolder/changeling/H = holder
 		if (istype(H) && H.in_fakedeath)
 			return 1
-		return M.stat || M.getStatusDuration("paralysis")
+		return !can_act(M, 0)
```
Comment (Tarmunora) [inline code/modules/antagonists/changeling/abilities/changeling.dm:336]: mechanical change

File: code/obj/critter/bee.dm [hunk 32]
```diff
@@ -1899,7 +1899,7 @@
 		src.visible_message("<span class='alert'><B>[src]</B> bites [M]!</span>")
 		logTheThing("combat", src.name, M, "bites [constructTarget(M,"combat")]")
 		random_brute_damage(M, 2, 1)
-		if (M.stat || M.getStatusDuration("paralysis"))
+		if (!can_act(M, 0))
```
Comment (Tarmunora) [inline code/obj/critter/bee.dm:1902]: mechanical change

File: code/obj/item/uplinks.dm [hunk 33]
```diff
@@ -284,7 +284,7 @@ Note: Add new traitor items to syndicate_buylist.dm, not here.
 		return 0
 
 #define CHECK1 (get_dist(src, usr) > 1 || !usr.contents.Find(src) || !isliving(usr) || iswraith(usr) || isintangible(usr))
-#define CHECK2 (usr.getStatusDuration("stunned") > 0 || usr.getStatusDuration("weakened") || usr.getStatusDuration("paralysis") > 0 || !isalive(usr) || usr.restrained())
+//#define CHECK2 (usr.getStatusDuration("stunned") > 0 || usr.getStatusDuration("weakened") || usr.getStatusDuration("paralysis") > 0 || !isalive(usr) || usr.restrained())
```
Comment (Tarmunora) [inline code/obj/item/uplinks.dm:287]: excise

File: code/obj/item/uplinks.dm [hunk 34]
```diff
@@ -379,7 +379,7 @@ Note: Add new traitor items to syndicate_buylist.dm, not here.
 		src.attack_self(usr)
 		return
 #undef CHECK1
-#undef CHECK2
+//#undef CHECK2
```
Comment (Tarmunora) [inline code/obj/item/uplinks.dm:382]: excise

File: code/WorkInProgress/cryospawn.dm (hunk 21 above)
Comment (Tarmunora) [reply code/WorkInProgress/cryospawn.dm:195]: probably doesn't matter

File: code/WorkInProgress/HaineWhatever.dm (hunk 20 above)
Comment (Tarmunora) [reply code/WorkInProgress/HaineWhatever.dm:1188]: revert or justify

File: code/WorkInProgress/tug.dm (hunk 22 above)
Comment (Tarmunora) [reply code/WorkInProgress/tug.dm:13]: probably doesn't matter

File: code/WorkInProgress/tug.dm (hunk 23 above)
Comment (Tarmunora) [reply code/WorkInProgress/tug.dm:49]: probably doesn't matter

File: code/WorkInProgress/tug.dm (hunk 24 above)
Comment (Tarmunora) [reply code/WorkInProgress/tug.dm:243]: probably doesn't matter

File: code/datums/abilities/misc/hunter_gearspawn.dm (hunk 25 above)
Comment (Tarmunora) [reply code/datums/abilities/misc/hunter_gearspawn.dm:42]: probably doesn't matter, they can do this in a closet in maint anyways

File: code/datums/abilities/misc/werewolf_transform.dm (hunk 26 above)
Comment (Tarmunora) [reply code/datums/abilities/misc/werewolf_transform.dm:42]: revert or justify

File: code/datums/abilities/werewolf/werewolf_transform.dm (hunk 27 above)
Comment (Tarmunora) [reply code/datums/abilities/werewolf/werewolf_transform.dm:43]: revert or justify

File: code/datums/abilities/wizard/phaseshift.dm (hunk 28 above)
Comment (Tarmunora) [reply code/datums/abilities/wizard/phaseshift.dm:38]: revert

File: code/datums/abilities/wizard/teleport.dm (hunk 29 above)
Comment (Tarmunora) [reply code/datums/abilities/wizard/teleport.dm:25]: revert

File: code/mob/melee_attack_procs.dm (hunk 30 above)
Comment (Tarmunora) [reply code/mob/melee_attack_procs.dm:482]: revert

File: code/modules/antagonists/changeling/abilities/changeling.dm (hunk 31 above)
Comment (Tarmunora) [reply code/modules/antagonists/changeling/abilities/changeling.dm:336]: revert or justify

File: code/modules/medical/surgery_procs.dm (hunk 2 above)
Comment (Tarmunora) [reply code/modules/medical/surgery_procs.dm:59]: revert

File: code/obj/critter/bee.dm (hunk 32 above)
Comment (Tarmunora) [reply code/obj/critter/bee.dm:1902]: probably doesn't matter

File: code/obj/item.dm (hunk 4 above)
Comment (Tarmunora) [reply code/obj/item.dm:636]: test this. PDAs etc should still be usable when stunned but not paralyzed by clickdragging to self I think - does this break that functionality?

File: code/obj/item/brain.dm (hunk 5 above)
Comment (Tarmunora) [reply code/obj/item/brain.dm:74]: Revert

File: code/obj/item/chem_pill_bottle.dm (hunk 6 above)
Comment (Tarmunora) [reply code/obj/item/chem_pill_bottle.dm:123]: probably doesn't matter

File: code/obj/item/clothing/gimmick.dm (hunk 7 above)
Comment (Tarmunora) [reply code/obj/item/clothing/gimmick.dm:77]: probably doesn't matter

File: code/obj/item/generic_box.dm (hunk 8 above)
Comment (Tarmunora) [reply code/obj/item/generic_box.dm:343]: probably doesn't matter

File: code/obj/item/gun/kinetic.dm (hunk 9 above)
Comment (Tarmunora) [reply code/obj/item/gun/kinetic.dm:1226]: probably doesn't matter

File: code/obj/item/misc_wizard_stuff.dm (hunk 10 above)
Comment (Tarmunora) [reply code/obj/item/misc_wizard_stuff.dm:43]: revert or justify

File: code/obj/item/tool/weldingtool.dm (hunk 11 above)
Comment (Tarmunora) [reply code/obj/item/tool/weldingtool.dm:82]: surgery check, revert

File: code/obj/mopbucket.dm (hunk 12 above)
Comment (Tarmunora) [reply code/obj/mopbucket.dm:55]: probably doesn't matter

File: code/obj/rack.dm (hunk 13 above)
Comment (Tarmunora) [reply code/obj/rack.dm:60]: probably doesn't matter

File: code/obj/storage/large_storage_parent.dm (hunk 14 above)
Comment (Tarmunora) [reply code/obj/storage/large_storage_parent.dm:299]: probably doesn't matter

File: code/obj/table.dm (hunk 15 above)
Comment (Tarmunora) [reply code/obj/table.dm:324]: probably doesn't matter

File: code/z_adventurezones/lavamoon.dm [hunk 35]
```diff
@@ -577,7 +577,7 @@ var/sound/iomoon_alarm_sound = null
 		src.attacking = 1
 		src.visible_message("<span class='alert'><B>[src]</B> pinches [M] with its claws!</span>")
 		random_brute_damage(M, 3,1)
-		if (M.stat || M.getStatusDuration("paralysis"))
+		if (!can_act(M, 0))
```
[the remark this answers is not in the document]
Comment (Tarmunora) [reply code/z_adventurezones/lavamoon.dm:581]: probably doesn't matter

**Review by Tarmunora — CHANGES_REQUESTED** [review]

**Comment by Tarmunora** [comment]
re surgery checks: that would let you debrain someone with a flash/stunbaton/taser, which can only end poorly imo
wizard phaseshift/teleport: I'm going to need serious convincing that preventing wizards from using their get-me-out-of-heres while stunned is a good idea
changeling powers / ww transform: similar concerns to above

**Comment by BatElite** [comment]
This branch is so far behind (IIRC months ago some tooling changes made it really annoying) that this PR is never getting updated.

May revisit the idea sometime, but do it in chunks rather than all at once.

**Pull request closed without merging**